### DISCUSSION TITLE
Github education discount link not opening

### DISCUSSION THREAD

**Discussion by abdelillahbel** [community/community#143109] [GitHub Education]
### I hope you're doing well!
I am facing a problem while entering the github education discount link, and it forward me to pricing page, this happens after the first year program end.

#### Thank you!

**Comment by abdelillahbel** [comment]
https://education.github.com/discount_requests/application

**Comment by davevad93** [comment]
Hi @abdelillahbel ,

I have seen this issue also happening to other people, contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by abdelillahbel** [reply]
I have opened many tickets, but after few minutes the ticket closed automatically without any reply ot response from support, and some times the bot reply and close it automatically.

**Reply by davevad93** [reply]
I see @abdelillahbel , the link I provided above is for actual support not the typical “bot support”, if you haven't opened a ticket yet, give it a try, but be aware that they can take a while to reply, even weeks.

**Reply by abdelillahbel** [reply]
I think I should contact a support with another way, I tried contacting support using a ticket and not the troubleshooting bot.
